### ISSUE TITLE
US dollar based pricing unfit for global token sales

### ISSUE THREAD

**Issue by miohtama** [saft-project/saft#1]
The current document discuss about US dollars. The most contemporary token sales price tokens in ETH or Bitcoin. Payments are in cryptocurrency, not in US bank wires. This makes the agreement as is unfit for most global token sales.

My suggestion is to not to include the exchange rate or US dollar discussion in the terms, as this might create false investor expectations as they expect US dollar based token sales or pricing when in fact pricing happens using a cryptocurrency.

<img width="539" alt="image" src="https://user-images.githubusercontent.com/49922/31083342-2020a328-a79a-11e7-8a8b-04466eae54c7.png">

<img width="530" alt="image" src="https://user-images.githubusercontent.com/49922/31083391-3f326cc4-a79a-11e7-8cca-d13fdabb0b33.png">

**Comment by jbenet** [comment]
Agreed! good points, all.

**Comment by Luckydee** [comment]
Agreed to a point,  but the current volatility in cryptocurrency,  need one to peg the contribution in fiat, so as to determine the cost for future reference. For example if an ico raise 20,000 ETH in 2016, will it be fair to still say an ICO which raise  20,000 ETH in 2017, both raise same amount of cryptocurrency.
